deploy/update-stack.sh

```
@@ -0,0 +1,213 @@
+      "$agent" "$(basename "$decl.pre-$AFTER-bak")"
+  else
+    rollback "$agent refuses and this script will not guess the fix: $verdict"
+  fi
+done
+
+# -------------------------------------------------------------------- 9. verify
+# A load that is not read back is an install that was not verified. This reads
+# the event out of the agent's own sink, the only place the claim can be
+# checked from.
+say "verify"
+AGENT=$(echo $ALLOW_LIST | awk '{print $1}')
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)